webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js

```diff
@@ -6,6 +7,27 @@ import {
 import { translate as __ } from '../../common/I18n';
 import { removeLastSlashFromPath } from '../../common/helpers';
 
+export const checkBrowserSupport = () => {
+  browserUpdate({
+    required: {
+      i: 12, // obsolete IE
+      e: -2,
+      f: -3,
+      o: -3,
+      s: -1,
+      c: -3,
+    },
+    text: __(
+      `<b class="buorg-mainmsg">We will drop support for your browser ({brow_name}) soon</b>.<br />
+      <span class="buorg-moremsg">Please update to modern browser, in case it is not possible for you, let us know you need to continue to support the browser.</span>
+      <span class="buorg-buttons"><a href="https://forms.gle/VL8H5Cby2LWZMhn69">Let us know</a> or <a{up_but}>update</a> or <a{ignore_but}>ignore</a></span>.`
```

Comment:
If we plan to link to a survey here, imho we should add some background or at least the project name once you hit this URL.

Reply:
Should we link to discourse?

Reply:
I have changed the title and linked discourse.